Makefile

```diff
@@ -408,6 +424,9 @@ test-ci-js: | clear-stalled
 		echo $${PS_OUT} | xargs kill; exit 1; \
 	fi
 
+# Run by most CI, exceptions:
```

Comment:
I think rpi is the _only_ exception, I've just been grepping through jenkins config files and it's the only one I can find.